base/src/main/java/com/smartdevicelink/proxy/rpc/VideoStreamingCapability.java

```diff
@@ -180,4 +184,14 @@ public VideoStreamingCapability setScale(Double scale) {
         setValue(KEY_SCALE, scale);
         return this;
     }
+
+    @SuppressWarnings("unchecked")
+    public List<VideoStreamingCapability> getAdditionalVideoStreamingCapabilities(){
+        return (List<VideoStreamingCapability>) getObject(VideoStreamingCapability.class, KEY_ADDITIONAL_VIDEO_STREAMING_CAPABILITIES);
+    }
+
+    public VideoStreamingCapability setAdditionalVideoStreamingCapabilities(List<VideoStreamingCapability> capabilities) {
```

Comment:
@kostyaBoss on line 193 you can change the param name to `additionalVideoStreamingCapabilities` (code generator tool).